geps/gep-1897.md

```diff
@@ -125,7 +125,231 @@ This GEP is the outcome of the TLS use cases #4 and #5 in
+| HTTPRoute  | Listener tls               | Yes                                           | **Yes**                       |
+| HTTPRoute  | No listener tls            | Yes                                           | **Yes**                       |
+| HTTPRoute  | Listener tls               | No                                            | No                            |
+| HTTPRoute  | No listener tls            | No                                            | No                            |
+| TLSRoute   | Listener Mode: Passthrough | Yes                                           | No                            |
+| TLSRoute   | Listener Mode: Terminate   | Yes                                           | Not supported                 |
+| TLSRoute   | Listener Mode: Passthrough | No                                            | No                            |
+| TLSRoute   | Listener Mode: Terminate   | No                                            | No                            |
+| TCPRoute   | Listener TLS               | Yes                                           | Not supported                 |
+| TCPRoute   | No listener TLS            | Yes                                           | Not supported                 |
+| TCPRoute   | Listener TLS               | No                                            | No                            |
+| TCPRoute   | No listener TLS            | No                                            | No                            |
+| UDPRoute   | Listener TLS               | N/A                                           | No                            |
+| UDPRoute   | No listener TLS            | N/A                                           | No                            |
+| UDPRoute   | Listener TLS               | N/A                                           | No                            |
+| UDPRoute   | No listener TLS            | N/A                                           | No                            |
+| GRPCRoute  | N/A                        | N/A                                           | No                            |
+
+## Request Flow
+
+One additional step would be added to the typical client/gateway API request flow for a gateway implemented using a
+reverse proxy. This is shown as step 6 below.
+
+1. A client makes a request to http://foo.example.com.
+2. DNS resolves the name to a Gateway address.
+3. The reverse proxy receives the request on a Listener and uses the Host header to match an HTTPRoute.
+4. Optionally, the reverse proxy can perform request header and/or path matching based on match rules of the HTTPRoute.
+5. Optionally, the reverse proxy can modify the request, i.e. add/remove headers, based on filter rules of the HTTPRoute.
+6. __(New) Optionally, the reverse proxy can determine the outcome of verifying the cert served by the backend, based on
```

Comment:
nit: its a bit confusing to say we are verifying the cert served, but not mentioning that we are adding TLS in the first place